org.eclipse.jdt.core.tests.model/src/org/eclipse/jdt/core/tests/dom/ASTConverterAST8Test.java

```diff
@@ -6548,7 +6549,7 @@ public void test0278() throws JavaModelException {
 		assertTrue("Not a type literal", expression instanceof TypeLiteral); //$NON-NLS-1$
 		ITypeBinding typeBinding = expression.resolveTypeBinding();
 		assertNotNull("No type binding", typeBinding); //$NON-NLS-1$
-		assertEquals("Wrong name", "Class", typeBinding.getName()); //$NON-NLS-1$ //$NON-NLS-2$
+		assertEquals("Wrong name", "Class<String>", typeBinding.getName()); //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
what caused this change? In this test class no compliance changes should apply (except where explicitly stated)?

Reply:
I assume the change in `ConverterTestSetup.getConverterJCLPath()` and in `ConverterTestSetup.setUpJCLClasspathVariables(String, boolean)` where both use `converterJclMin1.8.jar` now (`Class` is now `Class<T>` in the library) plus compilation settings change in 
`org.eclipse.jdt.core.tests.dom.ConverterTestSetup.setUpSuite()` that forces 1.8 instead of 1.4 JLS:
`setUpJavaProject("Converter", CompilerOptions.getFirstSupportedJavaVersion());`